### PR TITLE
Add operator support for QPoint and QPointF. 

### PR DESCRIPTION
**Pull request by KmolYuan**

### CODE REVIEW

File: PyQt5-stubs/QtCore.pyi [hunk 1]
```diff
@@ -5994,6 +5994,10 @@ class QPoint(sip.simplewrapper):
 
     def __neg__(self) -> 'QPoint': ...
     def __pos__(self) -> 'QPoint': ...
+    def __add__(self, point: 'QPoint') -> 'QPoint': ...
+    def __sub__(self, point: 'QPoint') -> 'QPoint': ...
+    def __mul__(self, point: 'QPoint') -> 'QPoint': ...
```
Comment (The-Compiler) [inline PyQt5-stubs/QtCore.pyi]: Multiplying a point by a point doesn't make any sense:

```
>>> QPoint(1, 2) * QPoint(1, 2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for *: 'QPoint' and 'QPoint'
```

This should probably be `factor: float` or so?

File: PyQt5-stubs/QtCore.pyi [hunk 2]
```diff
@@ -5994,6 +5994,10 @@ class QPoint(sip.simplewrapper):
...
+    def __truediv__(self, point: 'QPoint') -> 'QPoint': ...
```
Comment (The-Compiler) [inline PyQt5-stubs/QtCore.pyi]: Why `__truediv__` rather than `__div__`? Also, same as above.

File: PyQt5-stubs/QtCore.pyi [hunk 3]
```diff
@@ -6019,6 +6023,10 @@ class QPointF(sip.simplewrapper):
 
     def __neg__(self) -> 'QPointF': ...
     def __pos__(self) -> 'QPointF': ...
+    def __add__(self, point: 'QPointF') -> 'QPointF': ...
+    def __sub__(self, point: 'QPointF') -> 'QPointF': ...
+    def __mul__(self, point: 'QPointF') -> 'QPointF': ...
+    def __truediv__(self, point: 'QPointF') -> 'QPointF': ...
```
Comment (The-Compiler) [inline PyQt5-stubs/QtCore.pyi]: same as above for `__mul__` and `__truediv__`.

**Review by The-Compiler — CHANGES_REQUESTED** [review]

File: PyQt5-stubs/QtCore.pyi (hunk 1 above)
Comment (KmolYuan) [reply PyQt5-stubs/QtCore.pyi]: Sorry, it a typo. Float is right.

File: PyQt5-stubs/QtCore.pyi (hunk 2 above)
Comment (KmolYuan) [reply PyQt5-stubs/QtCore.pyi]: I think use Python 3 operator (`__truediv__` and `__floordiv__`) will be better than Python 2 (`__truediv__` and `__div__`).

Comment (The-Compiler) [reply PyQt5-stubs/QtCore.pyi]: Ah, true, I read the docs wrong there. My bad!

**Review by The-Compiler — APPROVED** [review]

**Review by stlehmann — APPROVED** [review]

**Pull request merged**